docs/SOPs/dataset_wrangling_SOP.md

```diff
@@ -124,30 +124,34 @@ Once you have a customised and potentially pre-filled spreadsheet it can be sent
 
 ### AWS User for Contributors
 
-In order for a contributor to upload their data, they would need their own AWS user that is assigned to the hca-contirbutor group. 
-A request for an account should be files as [a new ticket](https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/new?assignees=&labels=operations&template=new-contributor-account.md&title=contributor+account+for%3A+%3Ccontributor-name%3E) for the ingest team. The board is monitored regularly so the new ticket would be picked up within the day.
+In order for a contributor to upload their data, they would need their own AWS user that is assigned to the hca-contirbutor group.
 
-### Data upload Procedure
-These commands can be run by team members with "develoepr" role.
-- create an AWS user for them
+_**This is done by a team member with a developer role.**_
+
+1. create an AWS user for them using the name part of their email address
   
 ```shell
-# to add user alice as a contributor
-aws iam create-user --user-name alice --tags Key=project,Value=hca Key=owner,Value=tburdett Key=service,Value=ait
-aws iam add-user-to-group --group hca-contributor --user-name alice
+# to add user walter.white@example.com as a contributor
```

Comment:
I would add explicitly that we use the email name as the account name, and give the example you provided.

Reply:
I refined it.